### ISSUE TITLE
[Bug] Stocks/load autocomplete missing everything except file

### ISSUE THREAD

**Issue by jmaslek** [OpenBB-finance/OpenBBTerminal#4425]
Title
<img width="287" alt="Screenshot 2023-03-07 at 1 58 10 PM" src="https://user-images.githubusercontent.com/18151143/223523695-fe4a8b48-53e7-4460-a0bd-f2c306f95c22.png">

**Comment by deeleeramone** [comment]
In the same vein, `load --help` does not work.

```
2023 Mar 07, 17:50 (🦋) /stocks/ $ load --help

usage: load -t TICKER [-s START] [-e END] [-i {1,5,15,30,60}] [-p] [-f FILEPATH] [-m] [-w] [--exchange] [--performance] [-h] [--export EXPORT]
            [--sheet-name SHEET_NAME [SHEET_NAME ...]] [--source {YahooFinance,AlphaVantage,Polygon,EODHD,Intrinio,DataBento}]
load: error: the following arguments are required: -t/--ticker

2023 Mar 07, 17:50 (🦋) /stocks/ $ load --help

usage: load -t TICKER [-s START] [-e END] [-i {1,5,15,30,60}] [-p] [-f FILEPATH] [-m] [-w] [--exchange] [--performance] [-h] [--export EXPORT]
            [--sheet-name SHEET_NAME [SHEET_NAME ...]] [--source {YahooFinance,AlphaVantage,Polygon,EODHD,Intrinio,DataBento}]
load: error: the following arguments are required: -t/--ticker
```

**Comment by colin99d** [comment]
I fixed @deeleeramone's issue in #4504
